src/marqo/core/vespa_index/vespa_index.py

```diff
@@ -213,6 +213,19 @@ def _get_hybrid_score_modifiers(self, hybrid_query: MarqoHybridQuery) -> \
 
         return result
 
+    def escape(self, s: str) -> str:
+        """
```

Comment:
Is this method used for generating a lexical search query, a filter string, or both?

Reply:
Filter string only. We don't do this for lexical search, the mechanism is different.

Reply:
Got it. Can you add this to the doc string?